src/edgeworkers/client-manager.ts

```diff
@@ -1,4 +1,4 @@
-import * as cliUtils from '../../utils/cli-utils';
```

Comment:
Just curious, why are we removing the service directory?  Not a problem, just curious on the directory restructuring goals

Reply:
Originally in "master" branch they are not there. I added to branch "edgekv" when both edgeworkers and edgeKV services were under same directory. Now have moved them to their own directory and found it redundant to have them